db/silkworm/db/access_layer.cpp

```diff
@@ -35,11 +35,11 @@ std::optional<version_t> get_schema_version(mdbx::txn& txn) noexcept {
 
     auto data{src.current()};
     assert(data.value.length() == 12);
-    auto Major{boost::endian::load_big_u32(data.value.byte_ptr())};
+    auto Major{boost::endian::load_big_u32(static_cast<unsigned char*>(data.value.iov_base))};
     data.value.remove_prefix(sizeof(uint32_t));
-    auto Minor{boost::endian::load_big_u32(data.value.byte_ptr())};
+    auto Minor{boost::endian::load_big_u32(static_cast<unsigned char*>(data.value.iov_base))};
     data.value.remove_prefix(sizeof(uint32_t));
-    auto Patch{boost::endian::load_big_u32(data.value.byte_ptr())};
+    auto Patch{boost::endian::load_big_u32(static_cast<unsigned char*>(data.value.iov_base))};
```

Comment:
Please be consistent ... as in all other cases use uint8_t

Reply:
OK